hw/top_earlgrey/BUILD

```diff
@@ -109,6 +109,36 @@ fpga_cw310(
     rsa_key = {"//sw/device/silicon_creator/rom_ext/keys/fake:rom_ext_test_private_key_0": "test_key_0"},
 )
 
+# FPGA configuration used to emulate silicon targets. This rule can be used by
+# targets that require executing at the `rom_ext` stage level in flash, as well
+# as SRAM programs. Use the `fpga_cw310_sival_rom_ext` execution environment to
+# emulate silicon targets containing a `rom_ext` stage.
+fpga_cw310(
+    name = "fpga_cw310_sival",
+    testonly = True,
+    base = ":fpga_cw310_rom_with_fake_keys",
+    exec_env = "fpga_cw310_sival",
+
+    # PROD keys are allowed to run across all life cycle stages. This prod key
+    # is used to enable execution across life cycle stages with a single
+    # binary.
+    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:prod_private_key_0": "prod_key_0"},
+)
+
+# FPGA configuration used to emulate silicon targets containing a `rom_ext`
+# stage. See `fpga_cw310_sival` for more details.
+fpga_cw310(
+    name = "fpga_cw310_sival_rom_ext",
+    testonly = True,
+    base = ":fpga_cw310_rom_ext",
+    exec_env = "fpga_cw310_sival_rom_ext",
+
+    # PROD keys are allowed to run across all life cycle stages. This prod key
+    # is used to enable execution across life cycle stages with a single
+    # binary.
+    rsa_key = {"//sw/device/silicon_creator/rom/keys/fake/rsa:prod_private_key_0": "prod_key_0"},
```

Comment:
The specified RSA key is for the ROM.  I think we want this one instead:
```
rsa_key = {"//sw/device/silicon_creator/rom_ext/keys/fake:rom_ext_prod_private_key_0": "prod_key_0"},
```

Also, at the owner stage, the keys aren't bound to lifecycle states.  I don't know if there is a meaningful distinction between the test/dev/prod owner keys from a sival test point-of-view.

Reply:
Fixed. Yes, I am not sure if there is a meaningful distinction for firmware keys. I will leave that open for discussion for now.